public/js/application.js

```diff
@@ -1,7 +1,71 @@
 $(document).ready(function() {
-  // This is called after the document has loaded in its entirety
-  // This guarantees that any elements we bind to will exist on the page
-  // when we try to bind to them
+  $("#yield_wrapper").on("submit", "#new_question", function(e) {
+  	e.preventDefault();
+  	var request = $.ajax({
+  		type: "POST",
+  		url: e.target.action,
+  		data: $(this).serialize()
+  	});
+  	// @NYC_MENTORS Why did #reset not work?
+  	request.done(function(response) {
+  		$("#new_question").trigger("reset")
+  	});
+  	request.fail(function(response) {
+  		$("#error_box").text(response.responseText)
+  	});
+  });
+  $("body").on("submit", "#new_answer_form", function(e) {
+  		// console.log("yo")
+  		e.preventDefault();
+  		//console.log("ajax is working")
+  		var request = $.ajax({
+  			type: "POST",
+  			url: e.target.action,
+  			data: $(this).serialize()
+  		});
+  		request.done(function(response){
+  			// $("new_answer").trigger("reset")
+  			$("#answer_container").html(response);
+  		});
+  });
+  $(".container").hide()
```

Comment:
This container will likely flash up as the page loads. To prevent this set it to display none in the CSS.